scipy/stats/tests/test_stats.py

```diff
@@ -2120,6 +2122,7 @@ def test_regress_simple_onearg_cols(self):
         assert_almost_equal(result.stderr, 2.3957814497838803e-3)
         assert_almost_equal(result.intercept_stderr, 1.3866936078570702e-1)
 
+    @pytest.mark.filterwarnings('ignore::DeprecationWarning')
```

Comment:
I guess this one is tolerable, because the warning is subordinate to the error, and that whole test will go away once the deprecation is executed.